README.md

```diff
@@ -0,0 +1,228 @@
+- [About](#about)
+  * [Install](#install)
+  * [Example](#example)
+    + [Hello world with default AMQP library](#hello-world-with-default-amqp-library)
+    + [Same hello world with go-mq](#same-hello-world-with-go-mq)
+  * [Config in depth](#config-in-depth)
+  * [Error handling](#error-handling)
+  * [Epilogue](#epilogue)
+
+# About
+
+This package provides an ability to incapsulate creation and configuration [AMQP](https://www.amqp.org) entities 
+like queues, exchanges, producers and consumers in declarative way just with one config.
+
+Status: **WIP**.
+
+## Install
+
+`go get github.com/cheshir/go-mq`
+
+## Example
+
+### Hello world with default AMQP library
+
+```go
+package main
+
+import (
+	"github.com/streadway/amqp"
+)
+
+func main() {
+	conn, err := amqp.Dial("amqp://guest:guest@localhost:5672/")
+    if err != nil {
+        panic(err)
+    }
+    defer conn.Close()
+    
+    ch, err := conn.Channel()
+    if err != nil {
+        panic(err)
+    }
+    defer ch.Close()
+    
+    q, err := ch.QueueDeclare(
+        "hello_q", // name
+        true,    // durable
+        false,   // delete when unused
+        false,   // exclusive
+        false,   // no-wait
+        nil,     // arguments
+    )
+    if err != nil {
+        panic(err)
+    }
+    
+    err = ch.ExchangeDeclare(
+      "demo",   // name
+      "direct", // type
+      true,     // durable
+      false,    // auto-deleted
+      false,    // internal
+      false,    // no-wait
+      nil,      // arguments
+    )
+    
+    err = ch.QueueBind(q.Name, "route", "demo", false, nil)
+    if err != nil {
+        panic(err)
+    }
+    
+    body := "hello world!"
+    err = ch.Publish(
+        "demo", // exchange
+        "route", // routing key
+        false,  // mandatory
+        false,  // immediate
+        amqp.Publishing {
+          ContentType: "text/plain",
+          Body:        []byte(body),
+        },
+    )
+    if err != nil {
+        panic(err)
+    }
+}
+```
+
+Looks to verbose if you ask me.
+
+### Same hello world with go-mq
+
+Config file:
+
+```yaml
+mq:
+  dsn: "amqp://guest:guest@localhost:5672/"
+  exchanges:                # Exchange list.
+    demo:                   # Exchange name.
+      type: "direct"
+      options:
+      	durable: true
+  queues:                   # Queue list.
+    hello_q:                # Queue name.
+      exchange: "demo"      # Link to exchange "demo".
+      routing_key: "route"
+      options:
+        durable: true
+  producers:                # Producer list.
+    hello_p:                # Producer name.
+      exchange: "demo"      # Link to exchange "demo".
+      routing_key: "route"
+      options:
+        content_type: "text/plain"
+
+```
+
+And then:
+
+```go
+package main
+
+import (
+	"log"
+
+    "github.com/cheshir/go-mq"
+    "github.com/spf13/viper"
+)
+
+func main() {
+    // Read config. TODO in task #2.
```

Comment:
If you imply that the purpose of func main() is to reed config, you should write 
'Reads config'

Otherwise, if you are advising user to read config, leave it as is, 'Read config'.